src/systemd.rs

```diff
@@ -6,22 +6,26 @@ use std::{fs, thread};
 use color_eyre::eyre;
 use eyre::{eyre, Result, WrapErr};
 
-use crate::util::time::try_to_time;
+use crate::util::time::try_to;
```

Comment:
this import should also be limited to time such that usage reads like: `time::try_to`